src/.nuget/packages.config

```diff
@@ -3,4 +3,5 @@
   <package id="Microsoft.DotNet.BuildTools" version="1.0.25-prerelease-00040" />
   <package id="dnx-coreclr-win-x86" version="1.0.0-beta5-11580" />
   <package id="Microsoft.DotNet.BuildTools.ApiTools" version="1.0.0-prerelease" />
+  <package id="dnx-mono" version="1.0.0-beta5-11596" />
```

Comment:
Yeah, I'll probably do that. If we continue to have diverged dependencies perhaps we should create platform package configs for each platform. I'm going to assume that we will ultimately converge everything for now...